components/admin/admin-submission-status.tsx

```
@@ -69,30 +167,73 @@
         <div className="space-y-6">
           {/* Simple Pie Chart */}
           <div className="flex justify-center">
-            <div className="relative w-24 h-24 sm:w-32 sm:h-32">
-              <svg width="96" height="96" className="sm:w-32 sm:h-32 transform -rotate-90">
+            <div className="relative w-40 h-40 sm:w-48 sm:h-48">
+              <svg width="160" height="160" className="sm:w-48 sm:h-48 transform -rotate-90">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)